crates/revm/revm-inspectors/src/tracing/builder/parity.rs

```diff
@@ -165,10 +165,10 @@ impl ParityTraceBuilder {
         self,
         res: ResultAndState,
         trace_types: &HashSet<TraceType>,
-        db: DB,
+        db: &mut DB,
```

Comment:
is this right? should we be taking mutable reference on something that should be a read-only operation?

Reply:
It is fine